### DISCUSSION TITLE
Security Onion as a SOC-service

### DISCUSSION THREAD

**Discussion by hakfo** [Security-Onion-Solutions/securityonion#7634] [Unsupported Versions] [answered]
Hello everyone,

My name is Hakfo and me and my classmates are researching the possibility for creating a SOC-service that will manage several clients across our region. We have stumbled across Security Onion and found out about the extensive library of tools security onion provides which is really great. Our only concern with Security Onion is: Can we manage several different businesses/clients with a distributed deployment. To make it more clear: We are trying to build a security operation center that will monitors different businesses. We saw the architecture documents. But our question is this: Can we host our own manager node and seeker nodes in our own infrastructure, and then install forward nodes on different businesses their infrastructure, so that we can manage their logs and data. It would be really helpful if someone could give a solid answer to this question, since we can not find a similar use case on the internet that uses Security Onion to monitor several businesses.

Thanks in advance

^Hakfo

**Comment by dougburks** [comment, marked as answer]
> me and my classmates

If this is for a class, then I suspect your instructor would want you to do your own research. 

I'll point you to the [docs](https://securityonion.net/docs) and specifically the [Elastic license](https://docs.securityonion.net/en/2.3/license.html) as it may have an impact.

**Reply by hakfo** [reply]
Hello Doug,

Thank you for your reply. This is not specifically for a class or a school project, but this is purely for our own study/product research. We installed a standalone version already, so we know the possibilities of Security Onion. But since we can't find the feasibility for a large scale solution with several clients involved with Security Onion (there is also none in the docs), we tried to ask it on this discussion panel. We were hoping to find someone with experience for this use case that could explain us if Security Onion is feasible or not and what difficulties and problems will arise with our use case. Thanks for answering and we will be looking at the elastic search license costs thanks to your advice.
But we are still looking for someone with a lot of experience that could deep dive into our use case 😊

**Reply by hakfo** [reply]
We now see that it is not possible for our use case because of the license. Thanks for your answer!
